source/content/guides/collaborative-development.md

```diff
@@ -8,379 +8,335 @@ permalink: docs/guides/:basename
 
-* is only on Pantheon and you’re moving development to GitHub
-* exists on GitHub and you want to deploy to Pantheon
-* isn’t using Git
-* doesn’t yet exist
+- is only on Pantheon and you’re moving development to GitHub
+- exists on GitHub and you want to deploy to Pantheon
+- isn’t using Git
+- doesn’t yet exist
 
 ## Synchronizing Existing Pantheon Sites to GitHub
 
-From your Site Dashboard's Development environment, copy the Git SSH clone URL and clone it to your local machine. Your local copy will now track the Pantheon repository as origin.
+ 1. From your Site Dashboard's Development environment, copy the Git SSH clone URL and clone it to your local machine. 
 
-```bash{outputLines: 2-8}
-git clone <ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git d7-ci>
-Cloning into 'd7-ci'...
-remote: Counting objects: 298630, done.
-remote: Compressing objects: 100% (59210/59210), done.
-remote: Total 298630 (delta 208616), reused 298143 (delta 208186)
-Receiving objects: 100% (298630/298630), 67.23 MiB | 479.00 KiB/s, done.
-Resolving deltas: 100% (208616/208616), done.
-Checking connectivity... done.
-```
+  Your local copy will now track the Pantheon repository as origin.
 
-Change directory into the site repository, and verify your connection to the Pantheon server.
+  ```bash{outputLines: 2-8}
+  git clone <ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git d7-ci>
+  Cloning into 'd7-ci'...
+  remote: Counting objects: 298630, done.
+  remote: Compressing objects: 100% (59210/59210), done.
+  remote: Total 298630 (delta 208616), reused 298143 (delta 208186)
+  Receiving objects: 100% (298630/298630), 67.23 MiB | 479.00 KiB/s, done.
+  Resolving deltas: 100% (208616/208616), done.
+  Checking connectivity... done.
+  ```
+ 1. Change directory into the site repository, and verify your connection to the Pantheon server:
 
-```bash{outputLines: 2-4}
-cd d7-ci
-git remote -v
-origin  ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git (fetch)
-origin  ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git (push)
-```
+  ```bash{outputLines: 2-4}
+  cd d7-ci
+  git remote -v
+  origin  ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git (fetch)
+  origin  ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git (push)
+  ```
 
-The output lists "origin" as the remote with Pantheon SSH Git clone connection information as it's address.
+  The output lists "origin" as the remote with Pantheon SSH Git clone connection information as it's address.
 
 ### Create a Repository on GitHub
 
-![Create a Repository on GitHub](../../images/create-git-repo.png)
+1. Configure your GitHub respository to match the settings in the image below.
 
-### Add the GitHub Repository as a Remote
+ ![Create a Repository on GitHub](../../images/create-git-repo.png)
```

Comment:
@whitneymeredith - Are you able to retake this screenshot? It is rendering blurry 🤐

Reply:
@michellecolon-pantheon image replaced